fe/lib/components/BrandSelect/BrandSelect.tsx

```diff
@@ -80,8 +80,12 @@ export const BrandSelect = ({
 
       // Preselect the first brand from the query.
       setSelectedBrandId(firstBrandId);
+
+      if (onSelect) {
+        onSelect(firstBrandId);
+      }
```

Comment:
Are we allowed to do this:

```suggestion
      onSelect?.(firstBrandId);
```